test/Error/rlexe.xml

```diff
@@ -30,6 +30,8 @@
     <default>
       <files>stack.js</files>
       <baseline>stack.baseline</baseline>
+      <!-- xplat-todo: Understand why this test is failing on OS X -->
+      <tags>exclude_mac</tags>
```

Comment:
Direct link to the logs: http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/linux-ci/job/osx_osx_debug_static_prtest/10/